build.cake

```diff
@@ -0,0 +1,32 @@
+var target = Argument("target", "Build");
+
+Task("Clean")
+	.Does(() => 
+{
+	var settings = new DotNetCoreCleanSettings
+     {
+         Framework = "netstandard2.0",
+         Configuration = "Debug",
+         OutputDirectory = "./artifacts/"
+     };
+
+	 DotNetCoreClean("./src/CodingMilitia.PlayBall.Shared.StartupTasks", settings);
```

Comment:
Considering that this solution will have multiple projects, I don't know if it isn't easier in the long run to target the solution instead of the individual projects.

Reply:
YAGNI for now

Reply:
👍

Reply:
Just want to be sure that if someone else in the future grabs the script, doesn't start adding more projects manually instead of moving to a solution target.